tests/test_visitors/test_ast/test_annotations/test_return_annotations.py

```diff
@@ -58,6 +59,22 @@ def function() -> typing.Literal[1, 2, 3]: ...
 def function() -> Union[typing.Literal[2, 3], int]: ...
 """
 
+correct_union_with_none_return = """
+def function() -> Union[int, float, None]: ...
+"""
+
+correct_embedded_union_with_none_return = """
+def function() -> Union[Union[int, float, None], float]: ...
+"""
+
+correct_union_return = """
+def function() -> Union[True, float]: ...
```

Comment:
That's not correct. `True` it not a valid type. Let's use `Union[bool, float]`

Reply:
I just want to test if statement: `if isinstance(slice_arg, ast.NameConstant):` will it work with bool?

Reply:
Ok, I see. Then it is fine.